apps/ui/src/Pages/HomePage/HomePage.tsx

```diff
@@ -98,6 +104,25 @@ const HomePage: React.FC = () => {
             <ToggleButton value={'communities'}>My Communities</ToggleButton>
             <ToggleButton value={'rides'}>My Rides</ToggleButton>
           </ToggleButtonGroup>
+          {selectedTab === 'rides' && (
+            <Box sx={{ display: 'flex', alignItems: 'center', marginTop: '1rem' }}>
+              <ToggleButtonGroup
```

Comment:
It is not the best practice to create filters like this, because if we want to add more filters it will take too much space.
Please use the MUI Select component for the filter - https://github.com/CommuneCar/mono-app/pull/194#issuecomment-2169247785

Or, use a Switch like in the communities page - https://github.com/CommuneCar/mono-app/blob/c5598c908789be1b998570ddbb3b91e17f1e553e/apps/ui/src/Communities/MyEntitiesFilterButton.tsx

Reply:
@DarNachmani 
changed it to a switch button: 
<img width="407" alt="Screenshot 2024-06-17 at 12 49 20" src="https://github.com/CommuneCar/mono-app/assets/61973350/29b8581d-3771-4f20-8230-efb9d9603bb4">
<img width="407" alt="Screenshot 2024-06-17 at 12 49 20" src="https://github.com/CommuneCar/mono-app/assets/61973350/29b8581d-3771-4f20-8230-efb9d9603bb4">
<img width="407" alt="Screenshot 2024-06-17 at 12 49 20" src="https://github.com/CommuneCar/mono-app/assets/61973350/29b8581d-3771-4f20-8230-efb9d9603bb4">